cmd/admin-user-add-policy.go

```diff
@@ -44,6 +53,9 @@ FLAGS:
 EXAMPLES:
   1. Set a policy 'writeonly' to 'foobar' on MinIO server.
      $ {{.HelpName}} myminio foobar writeonly
+  2. Set a policy 'writeonly' to Kerberos principal 'mojo-jojo@MOJO.REALM' on myminio.
+     $ {{.HelpName}} myminio --kerberos mojo-jojo@MOJO.REALM writeonly
```

Comment:
```suggestion
      $ {{.HelpName}} myminio mojo-jojo@MOJO.REALM writeonly --sts-kerberos
```
cli flag needs to come at the end